sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -832,6 +832,17 @@ object SQLConf {
       .intConf
       .createWithDefault(10000)
 
+  val IGNORE_DATA_LOCALITY =
+    buildConf("spark.sql.sources.ignore.datalocality")
```

Comment:
conf naming looks a little weird...  Compared with the other SQL Confs, this should be renamed to `spark.sql.sources.ignoreDataLocality.enabled` cc @cloud-fan

Reply:
yea, please be careful about the namespace created in the new config names. `ignore` is definitely not a good namespace.

Reply:
@wangshisan Could you submit a follow-up PR to rename it?